### DISCUSSION TITLE
Linking of individual areas

### DISCUSSION THREAD

**Discussion by LarsVonwil** [requarks/wiki#6929] [Help / Questions]
Hello Wiki.js community
I have a question about the exit from the administration dashboard and when you click on the Wiki.js logo.

With these two variants you always end up on the page "Create new page or administration", is it possible to change this?

Many thanks for your help

**Comment by fnafox** [comment]
clicking on the Wiki.js logo leads you to the home page. it sounds like you don't have a home page that's why you're getting a prompt to create one.
